### PR TITLE
Create zoho_desk.json

### PR DESCRIPTION
**Pull request by datawisdomStudio**
Add Zoho Desk Service to the registry

### CODE REVIEW

**Comment by datawisdomStudio** [comment]
@googlebot I signed it!

**Comment by datawisdomStudio** [comment]
@googlebot I signed it!

**Comment by datawisdomStudio** [comment]
@googlebot I signed it!

File: sources/zoho_desk.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "id": "ZOHO_DESK",
+  "name": "Zoho DESK",
+  "categories": ["SOCIAL"],
```
Comment (pfrisella) [inline sources/zoho_desk.json:4]: perhaps CUSTOMER_SUPPORT or CRM are more appropriate?

**Review by pfrisella — CHANGES_REQUESTED** [review]

File: sources/zoho_desk.json (hunk 1 above)
Comment (datawisdomStudio) [reply sources/zoho_desk.json:4]: Agree, thanks!

**Review by pfrisella — APPROVED** [review]
LGTM. Thanks for updating.

**Pull request merged**